frontend/catalyst/utils/dummy.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Dummy module for testing"""
+
+
+def dummy_func(x):
+    """Simple function with if statements for testing"""
```

Comment:
@rauletorresc from what I know it is quite uncommon to have test files live in the production code. Maybe it is worth leaving more comments here, to explain to other developers what tests this is being used in?